plugins/module_utils/aci.py

```diff
@@ -1486,6 +1489,9 @@ def fail_json(self, msg, **kwargs):
                 self.result["sent"] = self.config
                 self.result["proposed"] = self.proposed
 
+        elif self.__class__.__name__ == "ACIRESTModule" and self.method in ["POST", "DELETE"]:
```

Comment:
Is possible but this assures it is only executed for ACIRESTModule class ( which is only used in aci_rest module). In case any other module has state not in self.params it would still execute which I want to avoid.